codecov/settings_base.py

```diff
@@ -252,7 +253,7 @@
         }
 
 # See https://django-postgres-extra.readthedocs.io/en/master/settings.html
-POSTGRES_EXTRA_DB_BACKEND_BASE: "django_prometheus.db.backends.postgresql"
+POSTGRES_EXTRA_DB_BACKEND_BASE: "django_prometheus.db.backends.postgresql"  # type: ignore
```

Comment:
Even post import it didn't resolve this correctly; but docs here: https://django-dbbackup.readthedocs.io/en/stable/databases.html